thirdparty/rax/radix.cc

```diff
@@ -2212,6 +2238,10 @@ std::string raxShow(rax *rax) {
     raxStr += "rax numnode:" + std::to_string(rax->numele) + "\n";
     raxStr += raxRecursiveShow(0,0,rax->head);
     raxStr += "\n";
+    // also show the backtrace info
+    std::stringstream ss;
+    backtrace_info::backtrace(ss, true);
+    raxStr += ss.str();
```

Comment:
Display back trace is costly, can you please wrap it into a `#ifndef _NDEBUG` block?

Reply:
Okay.